packages/common-all/src/constants/configs/commands.ts

```diff
@@ -165,6 +165,10 @@ const COPY_NOTE_LINK: DendronConfigEntryCollection<CopyNoteLinkConfig> = {
     label: "Non-note file",
     desc: "Copy note link settings for non-note files.",
   },
+  aliasMode: {
```

Comment:
I think we should make this name be specific to the `copyNoteLink` command - how about `copyNoteLinkAliasMode`?